lib/puppet/parser/functions/foreman.rb

```diff
@@ -36,6 +36,7 @@
 #
 # Happy Foreman API-ing!
 
+require "pry"
```

Comment:
if its not available on every installation , how we gonna solve it? maybe adding docs in https://theforeman.org/2012/01/getting-foreman-search-results-into.html ? but other side , as i know "pry" is only debugging so it cant effect for others who`s using this module. is it not?